front/cypress/e2e/content_builder/components/about_component.cy.ts

```diff
@@ -23,14 +23,12 @@ describe.skip('Content builder About component', () => {
       }).then((project) => {
         projectId = project.body.data.id;
         projectSlug = projectTitle;
-        cy.visit(`/admin/projects/${projectId}/description`);
-        cy.get('#e2e-toggle-enable-content-builder')
-          .find('input')
-          .click({ force: true });
+        cy.apiEnableContentBuilder({ projectId }).then(() => {
```

Comment:
Added a new Cypress command so we don't have to enable the content builder via the UI every time